README.md

```diff
@@ -1 +1,29 @@
 # Verse Eslint Configuration
+
+In an effort to keep the codebase consistent and easy to read, Verse projects should use a shared linting configuration. This repository exposes a shared configuration for the eslint tool. [eslint docs](http://eslint.org/)
+
+
+## Using the configuration file
+Assuming you have followed the getting started instructions on the eslint website and have a basic linting setup, follow the instructions to use the Verse version of the eslint configuration.
+
+* Install verse eslint configuration using npm 
+```
```

Comment:
Pro tip: since the code relates to the bullet point, indent the code by 4 spaces (1 tab) and the code will align to the bullets.